docs/api/proposed_endpoints.md

```diff
@@ -118,13 +118,14 @@ Status: 200 OK
     "first_name": "Jaun",
     "last_name": "Hawley",
     "email_id": "jaun@localhost.com",
-    "gender": "Male",
-    "tshirt_size": "Extra Large",
+    "gender": "male",
+    "tshirt_size": "extra large",
```

Comment:
Since these will be a choice field, we'll store them without spaces to refer them unambiguously.

Like `extra_large`